src/edit/index.vue

```diff
@@ -9,6 +9,9 @@
       active-placeholder="Use toolbar to upload the video"
       active-icon="mdi-arrow-up" />
     <div v-else>
+      <preview-overlay :show="shouldShowPreviewOverlay">
```

Comment:
Is `showPreviewOverlay` sufficient?

Reply:
I put that name at first, but then it sounded like a method name to me and I renamed it to `shouldShowPreviewOverlay` to sound more like a boolean variable. What do you think?